pkgs/tools/inputmethods/fcitx5/fcitx5-mozc.nix

```diff
@@ -0,0 +1,119 @@
+{ lib, clangStdenv, fetchFromGitHub, fetchurl, fetchpatch, fetchgit
+, python3Packages, mesa, ninja, pkg-config, protobuf, zinnia, qt5, fcitx5
+, jsoncpp, gtest, which, gtk2, unzip }:
+let
+  inherit (python3Packages) python gyp six;
+  japanese_usage_dictionary = fetchFromGitHub {
+    owner = "hiroyuki-komatsu";
+    repo = "japanese-usage-dictionary";
+    rev = "e5b3425575734c323e1d947009dd74709437b684";
+    sha256 = "0pyrpz9c8nxccwpgyr36w314mi8h132cis8ijvlqmmhqxwsi30hm";
+  };
+  # abseil-cpp in nixpkgs is too old
```

Comment:
Please update or overwrite it. There is https://github.com/NixOS/nixpkgs/pull/107846 to update it.

Reply:
#107846 seems to be a long term work, so I override it right here.

Reply:
Please overwrite the version in nixpkgs and use an newer version instead of vendoring it here.

Reply:
How can I do that? Could you please give some instructions?